src/ZendeskApi.Client/Resources/Ticket/TicketCommentsResource.cs

```diff
@@ -1,55 +1,49 @@
-using System;
 using System.Threading.Tasks;
```

Comment:
This file doesn't seem to be a part of the PR title? (DeletedTickets Resource). Should it be in this PR?